### DISCUSSION TITLE
Week 7

### DISCUSSION THREAD

**Discussion by ColinFay** [ColinFay/ur-first-5k#312] [General]
Hey y'all! 

Welcome to week 7 :) 

Here is this week menu: 

```
 ── Day 1 
  → W07D1-Cross Training
  Cross train, 30 minutes. Suggestions can be found at: https://github.com/ColinFay/ur-first-5k/discussions/109
  
  ── Day 2 
  → W07D2-Easy Run
  • Run, easy pace, 20 minutes.
  • Cool down, 5 to 10 minutes.
  • Stretch.
  
  ── Day 3 
  → Rest
  Rest day.
  
  ── Day 4 
  → W07D4-Steady Run
  • Run, easy pace, 5 minutes.
  • Run, steady pace, 5 minutes.
  • Run, easy pace, 5 minutes.
  • Run, steady pace, 5 minutes.
  • Cool down, 5 to 10 minutes.
  • Stretch.
  
  ── Day 5 
  → Rest
  Rest day.
  
  ── Day 6 
  → Rest
  Optional: Swap Sunday's run to today.
  
  ── Day 7 
  → W07D7-Easy Run
  • Run, easy pace, 30 to 40 minutes.
  • Cool down, 5 to 10 minutes.
  • Stretch.
```

Feel free to use this discussion thread if you have any question, suggestion, or if you just want to share your experience :) 

Enjoy ! 

Colin

**Comment by ColinFay** [comment]
@BenjaminLouis @fazepher @lewishounkpevi it seems that you haven't followed the program for the last 3 to 4 weeks, so I didn't include you inside this week plan.

Let me know if ever you still want to be in!

**Reply by fazepher** [reply]
Thanks for checking up on us. I'll try to get back on track starting today.

**Reply by BenjaminLouis** [reply]
Hey @ColinFay . Thanks for your message. I failed to be regular on your programm but I definitely use some of the session you offer from time to time. It's nice to have those !
